hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc

```diff
@@ -187,71 +180,116 @@ summed. If the processor has input edges with different priority, only
 edges with the highest priority will be reflected, after those are
 exhausted edges with the next lower priority will be reflected and so
 on.
-
-|`/proc=N` +
-&nbsp;&nbsp; `/ordinal=M`
-|This MBean groups metrics pertaining to processor instance _N_ and
-input or output edge _M_. _M_ can be a number, or it can be `snapshot`
-for output items written to state snapshot.
-
-* **topObservedWm**: The highest received watermark from _any_ input on
-edge _N_ +
-* **coalescedWm**: The highest watermark received from _all_ upstream
-processors on edge _N_ +
-
-* **emittedCount**: The number of emitted items. This number includes
-watermarks, snapshot barriers etc. Unlike `distributedItemsOut`, it
+|**proc=<N>, ordinal=<not specified>** +
+Each Jet member will have one instances of these metrics for each each processor
+instance _N_, the _N_ denotes the global processor index. Processor is the
+parallel worker doing the work of the vertex.
+
+|**topObservedWm**: The highest received watermark from _any_ input on
+edge _N_. +
+**coalescedWm**: The highest watermark received from _all_ upstream
+processors on edge _N_. +
+
+**emittedCount**: The number of emitted items. This number includes
+watermarks, snapshot barriers etc. Unlike _distributedItemsOut_, it
 includes items emitted items to local processors. +
-* **receivedCount**: The number of received items. This number does not
+**receivedCount**: The number of received items. This number does not
 include watermarks, snapshot barriers etc. It's the number of items the
-`Processor.process` method will receive. +
-* **receivedBatches**: The number of received batches.
-`Processor.process` receives a batch of items at a time, this is the
+_Processor.process_ method will receive. +
+**receivedBatches**: The number of received batches.
+_Processor.process_ receives a batch of items at a time, this is the
 number of such batches. By dividing _receivedCount_ by
 _receivedBatches_, you get the average batch size. It will be 1 under
 low load.
-
+|**proc=<N>, ordinal=<M>** +
+Each Jet member will have one instance of these metrics for each edge _M_ (input or
+output) of each processor _N_. _N_ is the global processor index and _M_ is either
+the ordinal of the edge or has the value _snapshot_ for output items
+written to state snapshot.
 |===
 
 === Hazelcast IMDG Metrics
```

Comment:
this may be worth moving up, or mentioning earlier since there is a lot of metrics from IMDG. We can give examples of what kind of metrics are available (i.e. map stats, connection stats etc)